apps/site/lib/site_web/views/event_view.ex

```diff
@@ -20,8 +20,15 @@ defmodule SiteWeb.EventView do
   def calendar_title(month), do: name_of_month(month)
 
   @spec no_results_message(String.t()) :: String.t()
-  def no_results_message(month) do
-    "There are no events in #{name_of_month(month)}."
+  def no_results_message(year) do
+    "There are no events in #{name_of_year(year)}."
+  end
```

Comment:
I believe we'll need a `no_results_message` function for both month and year.  Go to http://localhost:4001/events?month=2021-11-01, and it will say "There are no events in 2021." even though there are only no events in November.  OR will we redirect when a user goes to a month without events?

Reply:
Good question... Since we don't have much detail for how to handle empty states right now, I think I'm just going to remove this function completely for now.